manifests/init.pp

```diff
@@ -31,6 +31,11 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0644',
-    content => template('selinux/config.erb'),
+    content => template('selinux/config.erb');
```

Comment:
Please revert changes to this file. This is an addition of additional features not related to supporting EL 7. It also broke the build. By grouping your pull requests by topic it makes it easier for the maintainer to accept your code.